api_client/python/timesketch_api_client/client.py

```diff
@@ -1245,8 +1269,8 @@ def __init__(self, timeline_id, sketch_id, api, name=None,
             timeline_id: The primary key ID of the timeline.
             sketch_id: ID of a sketch.
             api: Instance of a TimesketchApi object.
-            name: Name of the timeline (optional)
-            searchindex: The Elasticsearch index name (optional)
+            name: Name of the timelne (optional)
+            searchindex: The Elasticsearch index name (optional)i
```

Comment:
same here

Reply:
(seems like you cut the "i" from timeline and pasted it at the end of this line

Reply:
seems like you added two comments to the same issue, both refer to https://github.com/deralexxx/timesketch/blob/patch-9/api_client/python/timesketch_api_client/client.py#L1272 --> fixed